packages/block-library/src/post-comment-content/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
```

Comment:
Let's maybe

```suggestion
import { comment as icon } from '@wordpress/icons';

```

here, and use it as `icon` further down.